src/pages/components/timeline.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Timeline
+description: >
+  The Timeline component lists events over a period of time.
+package: '@zendeskgarden/react-accordions'
+components:
+  - accordions/src/elements/timeline/Timeline.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To track an issue over time</li>
+      <li>To show events associated with an item</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To guide users through a step by step process, use a [Stepper](/components/stepper)
+          instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The timeline component is made up of individual timeline items. Item content appears to the right of the item marker by default.
+
+import TimelineDefault from '../../examples/accordions/timeline/TimelineDefault.tsx';
+import TimelineDefaultCode from '!!raw-loader!../../examples/accordions/timeline/TimelineDefault.tsx';
+
+<CodeExample code={TimelineDefaultCode}>
+  <TimelineDefault />
+</CodeExample>
+
+### Alternating
+
+Timeline items are displayed on alternating sides of the timeline.
```

Comment:
Suggest content: "Items are displayed on alternating sides of the Timeline." (note capitalization)